cmd/argocd/commands/account.go

```diff
@@ -80,14 +82,27 @@ func NewAccountUpdatePasswordCommand(clientOpts *argocdclient.ClientOptions) *co
 				errors.CheckError(err)
 			}
 
+			//Need to validate password complexity with regular expression
+			settingsConn, settingsIf, err := acdClient.NewSettingsClient()
+			defer io.Close(settingsConn)
+			errors.CheckError(err)
+			argoSettings, err := settingsIf.Get(context.Background(), &settingspkg.SettingsQuery{})
+			errors.CheckError(err)
+			passwordPattern := argoSettings.PasswordPattern
+			var validPasswordRegexp = regexp.MustCompile(passwordPattern)
```

Comment:
If the user configures an invalid regex then this line will panic. Instead we need to return error message